app/lib/frontend/templates/views/pkg/license.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:pub_dev/frontend/static_files.dart';
+
+import '../../../dom/dom.dart' as d;
+
+/// Nicely formatted labels for common SPDX identifiers.
```

Comment:
Why, is there not a quality to using identifiers? I don't really mind either way.

Reply:
I find it visually better (easier to read and parse) to have a space a lowercase starting letter in `BSD 3-clause` vs. `BSD-3-Clause`.

Reply:
For the search page results, I wouldn't put the raw identifiers there, as it is not indexed by Google, it is only read by humans.
For the package page, we could put the raw ids somewhere, maybe in a `title=""` tag perhaps?

Reply:
Right, what I meant was that of someone copies the text from the search results and then runs that through Google search, I feel more confident that they'll get the right page if the text uses the SPDX identifier.